app/src/main/res/values/strings.xml

```diff
@@ -306,6 +306,6 @@
 <string name="language_italian">Italian</string>
 <string name="language_portuguese">Portuguese</string>
 <string name="error_field_may_not_be_empty">This field may not be empty</string>
-<string name="error_value_out_of_range_0_to_255">Value is out of range (0-255)</string>
-<string name="error_value_out_of_range_0_to_65535">Value is out of range  (0-65535)</string>
+<string name="error_value_too_large_0_to_255">Value is out of range (0-255)</string>
```

Comment:
The "...out_of_range..." strings are the right ones. The "...value_too_large..." have been deprecated by my last commit in the PR.

Reply:
You are right. I just changed the value not the names in POEditor. My bad.